### PR TITLE
feat(browserContext): add BrowserContext.browser()

### PR DESCRIPTION
**Pull request by mxschmitt**
Talked with Pavel about that.

### CODE REVIEW

File: src/client/browserContext.ts [hunk 1]
```diff
@@ -92,6 +92,10 @@ export class BrowserContext extends ChannelOwner<channels.BrowserContextChannel,
     this._channel.setDefaultTimeoutNoReply({ timeout });
   }
 
+  browser(): Browser {
+    return this._browser!;
```
Comment (mxschmitt) [inline src/client/browserContext.ts:96]: not sure about the `!`. There should be always a browser instance.

Comment (yury-s) [reply src/client/browserContext.ts:96]: We should probably change the type of this field to `Browser` and throw in the constructor if `parent instanceof Browser` check fails.

File: test/browsercontext-basic.spec.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ it('should create new context', async function({browser}) {
   expect(browser.contexts().indexOf(context) !== -1).toBe(true);
   await context.close();
   expect(browser.contexts().length).toBe(0);
+  expect(browser).toBe(context.browser());
```
Comment (yury-s) [inline test/browsercontext-basic.spec.ts:29]: nit: also check before the context is closed?

**Review by yury-s — APPROVED** [review]

**Comment by mxschmitt** [comment]
> We should probably change the type of this field to Browser and throw in the constructor if parent instanceof Browser check fails.

When a browser was launched with a persistent context, then browser was undefined. Changed it to null, so it reflects to our current API design. (We don't expose undefined only null for nil elements).

**Pull request merged**